tests/test_convert.py

```diff
@@ -1,14 +1,11 @@
-"""
-Tests the Harmony convert module
-"""
+""" Tests the Harmony convert module """
```

Comment:
The changes in this test file and `test_adapter.py` are mainly:

* Only importing the necessary functions and classes from each package.
* Using TestCase.assertEquals in place of `assert variable_one == variable_two`.
* Fixing some indentation.
* Adding/removing whitespace as recommended by my linter.